sdcm/utils/remote_logger.py

```diff
@@ -291,7 +291,8 @@ class KubectlGeneralLogger(CommandNodeLoggerBase):
     def _logger_cmd(self) -> str:
         parent_cluster = self._node.parent_cluster
         cmd = parent_cluster.k8s_cluster.kubectl_cmd(
-            f"logs --previous=false -f --since={int(self.time_delta)}s", self._node.name, "-c",
+            f"logs --previous=false -f --since={int(self.time_delta)}s",
+            self.selector_or_name, "--tail=-1", "-c",
```

Comment:
What tail=-1 means ?

Reply:
```
kubectl logs --help
...
      --tail=-1: Lines of recent log file to display. Defaults to -1 with no selector, showing all log lines otherwise
10, if a selector is provided.
...
```

So, it is done to make the "-l %some-selector%" cases work in addition to the specific resource names usage.